lib/auth/webauthncli/fido2.go

```diff
@@ -323,6 +324,7 @@ func fido2Register(
 	}
 
 	rrk := cc.Response.AuthenticatorSelection.RequireResidentKey != nil && *cc.Response.AuthenticatorSelection.RequireResidentKey
+	log.Debugf("FIDO2: registration: rrk=%v", rrk)
```

Comment:
Nit: `rrk` would be hard to decypher for anyone not very familiar with this, how about `requireResidentKey=%v`?

Reply:
Done.